pkg/cloudprovider/providers/gce/gce_loadbalancer_external_test.go

```diff
@@ -75,10 +134,9 @@ func TestEnsureStaticIPWithTier(t *testing.T) {
 		},
 	} {
 		t.Run(desc, func(t *testing.T) {
-			ip, existed, err := ensureStaticIP(s, tc.name, serviceName, region, "", tc.netTier)
+			_, existed, err := ensureStaticIP(s, tc.name, serviceName, region, "", tc.netTier)
 			assert.NoError(t, err)
 			assert.False(t, existed)
-			assert.NotEqual(t, "", ip)
```

Comment:
Previously the [FakeCloudAddressService populated the IP addresses of any address added](https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/gce/gce_addresses_fakes.go#L64-L68). The GCE compute mock does not auto-populate the IP addresses. I opted to remove this check because it feels like it was just testing that the mock worked.